src/app/component/user/components/profile/edit-profile/personal-photo/personal-photo.component.ts

```diff
@@ -1,18 +1,55 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { ProfileService } from '../../profile-service/profile.service';
+import { Subscription } from 'rxjs';
+import { MatDialog } from '@angular/material/dialog';
+import { EditPhotoPopUpComponent } from '@shared/components/edit-photo-pop-up/edit-photo-pop-up.component';
+import { map } from 'rxjs/operators';
```

Comment:
Please, sort imports in such order: angular files, core files/libraries (RxJS etc.), local files

Reply:
fixed